pkg/ruler/rulestore/bucketclient/bucket_client.go

```diff
@@ -61,6 +62,11 @@ func (b *BucketRuleStore) getRuleGroup(ctx context.Context, userID, namespace, g
 		return nil, rulestore.ErrGroupNotFound
 	}
 
+	if userBucket.IsAccessDeniedErr(err) {
+		level.Debug(b.logger).Log("msg", "permission denied when loading group", "user", userID, "key", objectKey)
```

Comment:
Do we want to keep it debug level or use warn or something else?

Reply:
i think its fine as we will bubble up and log anyway on the ruler. WDYT?

Reply:
I am good with debug, just curious